soh/src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -607,38 +607,21 @@ void EnMag_DrawCharTexture(Gfx** gfxp, u8* texture, s32 rectLeft, s32 rectTop) {
     *gfxp = gfx;
 }
 
-s16 GetCharArraySize(const char* str) {
-    s16 length = 0;
-    char c = str[length];
-
-    while (c != 0) {
-        length++;
-        c = str[length];
-    }
-
-    return length;
-}
-
-static char* EnMag_GetCopyrightTex() 
-{
-    uint32_t gameVersion = ResourceMgr_GetGameVersion(0);
-
-    switch (gameVersion) {
-        case OOT_PAL_11:
-            return gTitleCopyrightN64Tex;
-        default:
-            return gTitleCopyrightGCTex;
-    }
-}
-
-static int EnMag_GetCopyrightTexWidth() {
-    uint32_t gameVersion = ResourceMgr_GetGameVersion(0);
-
+static void EnMag_SetCopyValues(const char** copy_tex, u16* copy_width, u16* copy_xl, u16* copy_xh) {
```

Comment:
it's not clear why this change is being made instead of utilizing the existing methods

Reply:
I had already done this for partial NTSC 1.0 support in the past, just copy the function for this new version.

Reply:
that works, i'm just generally quite risk averse with changes and this is moving us from a pattern that is tested and working to one that i had not seen tested before

Reply:
Check this: https://stackoverflow.com/questions/9144516/out-parameters-in-c

Reply:
i understand the pattern, that's not what i was taking issue with. it's the little things like moving to a defining a `const char*` and passing it by reference being a different pattern than using a returned `static char*` as a parameter directly. that change switches where in memory that string lives, which may have unintended consequences

Reply:
If you think it is convenient, we can use the previous method for this.

Reply:
the fact that you've tested it before with partial 1.0 support makes me not worried about it, i just wanted to explain where my hesitation was coming from